disco_aws_automation/disco_autoscale.py

```diff
@@ -91,13 +91,12 @@ def _get_group_generator(self, group_names=None):
             if not next_token:
                 break
 
-    def _get_instance_generator(self, instance_ids=None, hostclass=None, group_name=None):
-        '''Yields autoscaled instances in current environment'''
+    def _get_instance_generator(self, hostclass=None, group_name=None):
+        """Yields autoscaled instances in current environment"""
         next_token = None
         while True:
             instances = throttled_call(
-                self.connection.get_all_autoscaling_instances,
-                instance_ids=instance_ids, next_token=next_token)
+                self.connection.get_all_autoscaling_instances, next_token=next_token)
```

Comment:
We never pass a value when calling `get_all_autoscaling_instances` method, and in boto it defaults to None.